uber/tests/models/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from uber.tests import *
+
+
+@pytest.fixture
+def base_url(monkeypatch):
+    monkeypatch.setattr(c, 'URL_BASE', 'https://server.com/uber')
+
+
+def test_absolute_urls(base_url):
+    assert convert_to_absolute_url('../somepage.html') == 'https://server.com/uber/somepage.html'
+
+
+def test_absolute_urls_empty(base_url):
+    assert convert_to_absolute_url(None) == ''
+    assert convert_to_absolute_url('') == ''
+
+
+def test_absolute_url_error(base_url):
+    with pytest.raises(ValueError) as e_info:
+        convert_to_absolute_url('..')
```

Comment:
Purely a stylistic matter, but if you prefer a one-liner you can rewrite these two lines as

```
pytest.raises(ValueError, convert_to_absolute_url, '..')
```

The main reason to do the context manager thing is if you want to then make assertions about the exception instance that was raised, or something.  Though some people prefer it the way you have it just in terms of how the code looks.  I'm fine with it either way.